main.tf

```
@@ -80,6 +80,7 @@ resource "azurerm_template_deployment" "app_service_site" {
     env                  = "${var.env}"
     app_settings         = "${jsonencode(merge(local.production_slot_app_settings, var.app_settings_defaults, local.app_settings_evaluated, var.app_settings))}"
     staging_app_settings = "${jsonencode(merge(var.staging_slot_app_settings, var.app_settings_defaults, local.app_settings_evaluated, var.app_settings))}"
+    additional_host_names = "${var.additional_host_names}"
```

```suggestion
    additional_host_names = "${join(",", var.additional_host_names}"
```

may need to convert to null for empty or change the arm template to handle empty string instead of null